apps/frontend/src/types/opportunity.types.ts

```diff
@@ -30,7 +30,7 @@ export const opportunityStatusMap: Record<OpportunityStatus, string> = {
 };
 
 export interface OpportunityType {
-  id: string;
+  id?: string;
```

Comment:
por que dejaste este como opcional?

Reply:
Si lo hiciste asi porque te pide el id al crear la opportunidad, puedes usar omit 

![image](https://github.com/user-attachments/assets/d886da3f-8fca-4905-9b7e-6ab5e8a17650)

```suggestion
export const createOpp = (opp: Omit<OpportunityType, "id">) => { }
```

Reply:
me salía un error y no sabia como resolverlo xd. Ya resolvi, ya no tiene ?